tests/add-user-ui.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { test, expect } from "@playwright/test";
+import { Colors } from "../enums/Colors";
+import { generateRandomUserName } from "../helpers/generateRandomUserName";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+});
+
+test("Check that page 'Add User' has title - TS Trainee course", async ({
+  page,
+}) => {
+  await expect(page).toHaveTitle("TS Trainee course");
+});
+
+test("Verify 'Create' button design on the 'Add User' page", async ({
+  page,
+}) => {
+  const createBtn = page.locator("xpath=//div[4]/button");
+  await expect(createBtn).toBeVisible;
+  await expect(createBtn).toHaveCSS("background-color", Colors.lightBlue);
+  await createBtn.hover();
+  await expect(createBtn).toHaveCSS("background-color", Colors.darkBlue);
+});
+
+test("Verify 'Cancel' button design on the 'Add User' page", async ({
+  page,
+}) => {
+  const cancelBtn = page.getByRole("link", { name: "Cancel" });
+
+  await expect(cancelBtn).toHaveCSS("background-color", Colors.lightGrey);
+  await cancelBtn.hover();
+  await expect(cancelBtn).toHaveCSS("background-color", Colors.darkGrey);
+});
+
+test("Verify 'User Name' field design and maximum symbols limit on the 'Add User' page", async ({
```

Comment:
divide your tests into two ares. 
First will be related to design, other - to functional checks